src/InstallHelperPlugin.php

```diff
@@ -118,6 +118,11 @@ public function onWunderIoDdevDrupalPackageInstall(PackageEvent $event) {
       return NULL;
     }
 
+    // Nothing to do here anymore.
+    if (!getenv('IS_DDEV_PROJECT')) {
```

Comment:
I think maybe even add it to the beginng on the function. We don't need to call the above it's not DDEV.